benchmarks/cugraph/standalone/bulk_sampling/run_train_job.sh

```diff
@@ -28,13 +26,13 @@ mkdir -p $LOGS_DIR
 mkdir -p $SAMPLES_DIR
 mkdir -p $DATASETS_DIR
 
-BATCH_SIZE=512
+BATCH_SIZE=555
```

Comment:
Why did we choose `555`, seems arbitary ?

Reply:
It is arbitrary - that was a temporary number I used for an experiment.  It should be reverted to 512